Added introduction.md

d3crypt3d opened the pull request.

PeterBell committed:
Merge pull request #2 from githubteacher/master

Most Senseis should be respected

Commit:
Initial kill list

PeterBell committed:
Merge pull request #3 from deadlyvipers/kill_list

Initial kill list

Commit:
We dont need to kill Budd

Commit:
Added introduction.md

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1 @@
+Just my introduction.
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Commit:
Added favourite path

Commit:
Added deadly skills to introduction.md

Commit:
Added killing history

codeschool-kiddo committed:
Added new rule about coffee

Commit:
Fixed bugs in dojo rules

d3crypt3d committed:
Merge pull request #2 from DreamWalker87/new_rules

Too many people are spilling coffee

Commit:
merging deadly skills

ills'

# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.

Commit:
Updated kill list

Commit:
merged hotfix

d3crypt3d committed:
Added personal grievance, fixes #3

d3crypt3d committed:
Merge pull request #4 from DreamWalker87/kill_list

Added personal grievance, fixes #3

d3crypt3d committed:
Changed README.md
